### ISSUE TITLE
Default Port Problem

### ISSUE THREAD

**Issue by cRUSHER-HLG** [easy-wi/developer#1129]
My problem is that at the night the server with the port 27015 crashes and keep offline all the time. I need to stop all servers and then begin restart with the server on default port. Someone a idea why this is so? Logs checked and nothing found.

app_server.log:
Wed May 30 03:54:32 CEST 2018: Server template /home/user2-29/server/csgo/ owned by user user2-29 added/synced
Wed May 30 03:54:33 CEST 2018: App IP_PORT (Changed) owned by user user2-29 started

At the day all servers running only at night from 1 am it dont works clear. Server is writing the message above in seconds betweens.

Webhost: External Host
Root: Gaming Root Install

easywi logs also say that it see, that the server is offline: 
Log: Starten Gameserver ip:27015 (Found offline since 241 checks)

**Comment by cRUSHER-HLG** [comment]
Found something interessting. If you check which ports are running on the servers with netstat -tunlp then 3 of 4 of my servers only take 1 port. 27025, 27035, 27065. But the server on default port 27015 has also (27016, 27017, 27018, 27019). I have removed all ports on my server with the default port 27015 that this is the single port that he can use. If the server crash at night it dont starts on port 27015 he takes the next port 27016. Does someone know a solution/fix for the problem?
